scheduler/plugin/host.go

```diff
@@ -48,6 +49,8 @@ const (
 	k8sSchedulerResultNominatedNodeName   = "result.nominated_node_name"
 	k8sSchedulerResultStatusReason        = "result.status_reason"
 	k8sSchedulerResultNormalizedScoreList = "result.normalized_score_list"
+	k8sHandle                             = "k8s.io/handle"
+	k8sHandleEventRecorderEventf          = "eventrecorder.eventf"
```

Comment:
This should be in `k8sScheduler` module since `framework.Handle` is scheduler specific stuff rather than something in k8s in general.